### ISSUE TITLE
A11y_AzureDataStudioSQL_Project_CreateNewProject_Popup_Keyboard: Keyboard focus goes outside of the popup by hitting tab key

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#14304]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.3;#Keyboard;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.26.0-insider (user setup)
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab till "Create New Project" button or "+" button and hit enter.
4. Fill out every field and click "ok" button.
5. A popup will appear
6. Navigate inside the popup and hit tab keys and check where keyboard focus goes

### Actual:
Keyboard focus goes outside of the popup by hitting tab key

Issue observed with shift + tab key also.

### Expected:
Keyboard focus should not go outside of the popup by hitting tab key

### User Impact:
Keyboard only users will face difficulty in navigating again in the popup

**Comment by Shivamd1** [comment]
@kisantia Could you please let us know in which build the issue is fixed so that we can verify?
Also could you please reopen the bug so that i can close it after verifying?

**Comment by kisantia** [comment]
@Shivamd1 sure will do. I'll update the build and should be able to get it to you tomorrow

**Comment by kisantia** [comment]
@Shivamd1 build is now updated. Please test using the insiders build and version 0.8.0 of the sql database projects extension

**Comment by Shivamd1** [comment]
Verified the issue in the insider build and  version 0.8.0 of the sql database projects extension and with reference to this  #14741 the popup dialog opens where focus does not go outside hence closing this issue.
